files/grest/rpc/pool/pool_list.sql

```diff
@@ -8,19 +8,32 @@ CREATE FUNCTION grest.pool_list ()
   AS $$
   # variable_conflict use_column
 BEGIN
-  RETURN QUERY SELECT DISTINCT ON (pic.pool_id_bech32)
-    pool_id_bech32,
-    pod.ticker_name
-  FROM
-    grest.pool_info_cache AS pic
-  LEFT JOIN public.pool_offline_data AS pod ON pod.pmr_id = pic.meta_id
-WHERE
-  pic.pool_status != 'retired'
-ORDER BY
-  pic.pool_id_bech32,
-  pic.tx_id DESC;
-END;
-$$;
 
-COMMENT ON FUNCTION grest.pool_list IS 'A list of all currently registered/retiring (not retired) pools';
+  RETURN QUERY (
```

Comment:
can we change this to return table format? (consistency as well as unlocks full potential for PostgREST API)

PS: There is a seperate ticket for this request, so can be looked at seperately for almost all pool queries

Reply:
I am confused - the function is defined as `RETURNS TABLE`. I thought we have to `RETURN QUERY` in the actual function and `RETURN QUERY` actually returns a table?

Reply:
ah - my bad, I misread it